sonic-net/sonic-mgmt
# Add DASH VNET test cases

### Description of PR


Summary:
Fixes # (issue)

### Type of change





### Back port request

### Approach
Test cases:

- VNET outbound traffic:
    - Send outbound traffic from the VM VNI, verify that traffic is re-encapped correctly according to mapping table
- VNET inbound traffic with PA validation:
    - Send inbound traffic from a remote VNI with outer source IP as the PA in the VNET mapping table, verify traffic is re-encapped correctly and forwarded back to VM
    - Send inbound traffic from remote VNI with outer source IP different from the PA in the mapping table, verify traffic is dropped

#### What is the motivation for this PR?

#### How did you do it?

#### How did you verify/test it?

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
